docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -1,41 +1,108 @@
+# _*_ coding: utf-8 -*-
+# Aggregate the outputs gotten from crawling the documentation of
+# different ML libraries
+
 import json
 import re
 import warnings
 from translations import WORD_TRANSLATIONS, COMMAND_TRANSLATIONS
 
+# The Compiler class is responsible for handling every stage of the
+# aggregation. The Class starts by reading the Individual outputs of
+# each library, then If an Exisiting Aggregate exists, It loads the
+# Aggregate from translations.py. The final step is to then merge
+# the exisiting libraries, or update an exisitng Aggregate with new
+# functions or Libraries.
+
```

Comment:
These comments are great! Would it be possible to document them in the format that [Sphinx can use?](https://pythonhosted.org/an_example_pypi_project/sphinx.html#full-code-example)